packages/agreement-process/src/model/domain/validators.ts

```diff
@@ -222,14 +213,10 @@ export const verifySubmissionConflictingAgreements = async (
   agreement: Agreement,
   agreementQuery: AgreementQuery
 ): Promise<void> => {
-  const conflictingStates: AgreementState[] = [
```

Comment:
moved this to `models` as `agreementSubmissionConflictingStates` to reuse it in the unit test

Reply:
same